README.md

```diff
@@ -208,6 +209,37 @@ ros2 run topic_tools mux sel_cmdvel auto_cmdvel joystick_cmdvel
 - `initial_topic` (str, default="")
     - Input topic to select on startup. If `__none`, start with no input topic. If unset, default to first topic in arguments
 
+### Demux
+
+demux is a ROS2 node that subscribes to an incoming topic and republishes incoming data to one of many topic,
+i.e., it's a demultiplexer that switches an input towards 1 of N outputs. Services are offered to switch among output topics,
+and to add and delete output topics. At startup, the first output topic on the command line is selected.
+
+#### Usage
+
+```shell
+ros2 run topic_tools demux <intopic> <outtopic1> [outopic2...]
+```
+
+Subscribe to <intopic1> and publish currently to selected outopic among <outopic1>...N. demux will start with <outopic1> selected.
+- `intopic`: Incoming topic to subscribe to
+- `outopicN`: Outgoing topic to publish on
```

Comment:
We need to be consistent about naming. It is better to use `outtopic` with two `tt` or `out_topic` with an underscore.

Reply:
Fixed for mux and demux